### DISCUSSION TITLE
Use textcat and textcat_multilabel in the same model

### DISCUSSION THREAD

**Discussion by silvathales** [explosion/spaCy#10819] [Help: Coding & Implementations]
Hello!

I have a model that works with NER and TextCategorizer. 

Can TextCategorizer be trained using both textcat and textcat_multilabel?
I need to evaluate the relevance of a text (textcat) and add tags (textcat_multilabel).
Is it possible to do this using a single model?

thank you!

**Comment by polm** [comment]
You can do this with two instances of textcat in the same pipeline, one for each of your models. However you'd need to train them separately, since there's only one place to put labels for textcat on a document. 

After you've trained the components separately you can source one of them into the pipeline of the other. Take a look at the [double NER example project](https://github.com/explosion/projects/tree/v3/tutorials/ner_double) for something similar.

**Reply by polm** [reply]
Also take a look at #10177, which is kind of a similar problem.
